lib/python/gunittest/invoker.py

```diff
@@ -206,7 +206,7 @@ def _run_test_module(self, module, results_dir, gisdbase, location):
             shutil.rmtree(mapset_dir)
 
     def run_in_location(self, gisdbase, location, location_type,
-                        results_dir):
+                        results_dir, test_list_in, test_list_out):
```

Comment:
If anyone is calling `run_in_location()`, their code will be broken after this commit since they will only be giving 4 arguments instead of 6. Perhaps `test_list_in` and `test_list_out` could have some default values?

Reply:
True, now they have default values.